root-files/opt/flownative/lib/nginx-legacy.sh

```diff
@@ -209,7 +216,16 @@ EOM
         proxy_pass \$assetUri;
     }
 EOM
-
+    else
+        cat >>"${NGINX_CONF_PATH}/sites-enabled/site.conf" <<-EOM
+    location ~* ^/_Resources/Persistent/(.*)$ {
```

Comment:
Compared to above, why is this still hardcoded and not using a variable?

Reply:
This else case handles a setup without cloud storage, but storing persistent resources in a local file system. I mean, we possibly _can_ support other directories, for that, but does it make sense?